build.sh

```diff
@@ -1,15 +1,15 @@
-#!/bin/bash
+#!/bin/bash -eu
 
-set -eu
+# Make sure we are in the directory this script is in.
+cd "$(dirname "$0")"
 
-if [ ! -d build ]
+if [[ ! -d build ]]
```

Comment:
The bash variant `[[` is usually safer than the sh variant `[`. It does not matter here, but I try to consistently use `[[`.